wiki/Tournaments/CWC/2025/en.md

```diff
@@ -110,30 +111,18 @@ The osu!catch World Cup 2025 is run by various community members.
 | ::{ flag=US }:: | **United States** | **[Secre](https://osu.ppy.sh/users/2306637)**, [Dahcreeper](https://osu.ppy.sh/users/6926006), [Lexii](https://osu.ppy.sh/users/7226149), [Elux](https://osu.ppy.sh/users/12004983), [The 47th](https://osu.ppy.sh/users/4990320), [yeeeter](https://osu.ppy.sh/users/15274666) |
 | ::{ flag=VN }:: | **Vietnam** | **[Stardust Prism](https://osu.ppy.sh/users/8525921)**, [ARL](https://osu.ppy.sh/users/15778330), [mothernature](https://osu.ppy.sh/users/6403418), [NamSPro](https://osu.ppy.sh/users/11387006), [Shu](https://osu.ppy.sh/users/4744615) |
 
-The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/19a8df801dc4c99f192694621928b231).
-
-## Match schedule: Grand Finals
-
-### Saturday, 19 July 2025
-
-| Team A | Team B | Match time | Twitch stream |  |
-| --: | :-- | :-- | :-: | :-: |
-| Grand Finals | showmatch | [Jul 19 (Sat) 15:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20250719T150000&p1=1440) | [osulive](https://twitch.tv/osulive) | [^showmatch] |
+Captains are listed in **bold**.\The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/19a8df801dc4c99f192694621928b231).
 
-### Sunday, 20 July 2025
+## Podium
 
-| Team A | Team B | Match time | Twitch stream |  |
-| --: | :-- | :-- | :-: | :-: |
-| United States ::{ flag=US }:: | ::{ flag=FI }:: Finland | [Jul 20 (Sun) 16:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20250720T160000&p1=1440&p2=263&p3=101) | [osulive](https://twitch.tv/osulive) | [^lb-grand-finals] |
-| Argentina ::{ flag=AR }:: | ::{ flag=US }:: United States | [Jul 20 (Sun) 17:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20250720T170000&p1=1440&p2=51&p3=263) | [osulive](https://twitch.tv/osulive) | [^grand-finals] |
-| Argentina ::{ flag=AR }:: | ::{ flag=FI }:: Finland | [Jul 20 (Sun) 17:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20250720T180000&p1=1440&p2=51&p3=101) | [osulive](https://twitch.tv/osulive) | [^grand-finals] |
+![](img/cwc2025-podium.jpg "CWC 2025 podium")
 
 ## Mappools
 
 ### Grand Finals
 
-**[Download the mappack here (207 MB)](https://packs.ppy.sh/P293%20-%20osu!catch%20World%20Cup%202025%3A%20Grand%20Finals.zip)**\
-[View the showcase VOD here](https://www.twitch.tv/videos/2511672807?t=0h14m36s)
+**[Download the mappack here (207 MB)](https://packs.ppy.sh/P293%20-%20osu!catch%20World%20Cup%202025%3A%20Grand%20Finals.zip?1752690501)**\
+[Watch the showcase VOD here](https://youtu.be/wXIoAfJTXRw?t=14884)
```

Comment:
for future reference, i recommend using the full url (https://www.youtube.com/watch?v=wXIoAfJTXRw&t=14884s) to avoid having to wait (although briefly) for the redirect to resolve when clicking the link